app/frontend/src/components/designer/FormDesigner.vue

```diff
@@ -477,11 +677,18 @@ export default {
       this.fetchForm(this.formId);
     }
   },
+  mounted() {
+    if (!this.formId) {
+      // We are creating a new form, so we obtain the original schema here.
+      this.$options.originalSchema = JSON.parse(JSON.stringify(deepFreeze(deepClone(this.formSchema))));
+      deepFreeze(this.$options.originalSchema);
```

Comment:
If we are already deepCloning an object, there is no need to do a JSON stringify/parse chain as it is already doing the exact same thing. If I understand the intent of code, this can be a one-liner:

```js
this.$options.originalSchema = deepFreeze(deepClone(this.formSchema));
```

Reply:
Ah, the reason for this was because something kept making the originalSchema observable and changing it even though it should be read only. I think I found that it was occurring when generating the final schema in the getPatch method.